### PR TITLE
RSEF announcement

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: content/posts/matplotlib-rsef/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+[ObsPy](https://obspy.org/), and [NumPy](https://numpy.org/). Outside
+of Python, Elliott is a developer on the [Pidgin
+project](https://pidgin.im/) and a packager for [Fedora
+Linux](https://getfedora.org/). In working on Matplotlib, he is
+interested in promoting Free and Open Source software and advancing
+science through reproducible workflows and more accessible libraries.
+
+As has been discussed in
+[detail](https://www.fordfoundation.org/work/learning/research-reports/roads-and-bridges-the-unseen-labor-behind-our-digital-infrastructure/)
+[](http://ford)[elsewhere](https://numfocus.org/programs/sustainability)
```
Comment (QuLogic) [inline content/posts/matplotlib-rsef/index.md]: I think this empty link is a typo:
```suggestion
[elsewhere](https://numfocus.org/programs/sustainability)
```

File: content/posts/matplotlib-rsef/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+date: 2020-03-17T19:51-04:00
+draft: True
+description: "We have hired Elliott Sales de Andrade as the Matplotlib Software Research Engineering Fellow supported by the Chan Zuckerberg Initiative Essential Open Source Software for Science"
+categories: ["Announcement"]
+author: Thomas A Caswell, Hannah Aizenman, Elliott Sales de Andrade
+displayInList: true
+---
+
+
+We are happy to announce that we have hired Elliott Sales de Andrade ([qulogic](https://github.com/qulogic))
```
Comment (QuLogic) [inline content/posts/matplotlib-rsef/index.md]:
```suggestion
We are happy to announce that we have hired Elliott Sales de Andrade ([QuLogic](https://github.com/QuLogic))
```

Comment (tacaswell) [reply content/posts/matplotlib-rsef/index.md]: Sorry about that.

**Review by QuLogic — APPROVED** [review]
Don't forget to un-draft it when publishing (and change the date if you wish.)

**Comment by tacaswell** [comment]
I flipped to not-draft, leaving to @QuLogic to push the button as final approval.

**Review by story645 — APPROVED** [review]

**Comment by QuLogic** [comment]
I don't have that power on this repo, but it LGTM.

**Pull request merged**